trello.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pprint
+import requests
+
+pp = pprint.PrettyPrinter(indent=4)
+app_board_name='To Do App'
+trello_member='brendanmcdaniel'
+trello_endpoint='https://api.trello.com/1/'
+
+class item:
+    def __init__(self, id, status, title):
+        self.id = id
+        self.status = status
+        self.title = title
+
+class status:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+# Check if proxy env vars have been set
+try:
+    http_proxy = os.environ['http_proxy']
+except KeyError:
+    pp.pprint('No HTTP proxy found')
+    http_proxy = ''
+try:
+    trello_key = os.environ['TRELLO_KEY']
+    trello_token = os.environ['TRELLO_TOKEN']
+except KeyError:
+    pp.pprint('Trello env vars dont seem to be set')
+    trello_key = ''
+    trello_token = ''
+
+# Environment variables set for trell api
+try:
+    https_proxy = os.environ['https_proxy']
+except KeyError:
+    pp.pprint('No HTTPS proxy found')
+    https_proxy = ''
+
+proxyDict = { 
+            "http"  : http_proxy, 
+            "https" : https_proxy
+            }
+
+headers={}
+
+def get_item(id):
+    tempItem = requests.get(trello_endpoint+'cards/'+id+'?key='+trello_key+'&token='+trello_token, headers=headers, proxies=proxyDict).json()
```

Comment:
General point for all of your API calls.

You could write this as:
```
auth_params = { 'key': trello_key, 'token': trello_token }
requests.get(f"{trello_endpoint}cards/{id}", headers=headers, params=params, proxies=proxyDict).json()
```

And all your functions could share auth_params, adding extra params if needed.